ci/jenkins/Jenkins_steps.groovy

```diff
@@ -862,6 +862,21 @@ def test_unix_clojure_cpu() {
     }]
 }
 
+
+def test_unix_clojure_cpu_nightly() {
```

Comment:
You can leave that out for now. I didn't yet have the chance to migrate nightly to the new design.
But thanks for early adopting!